src/Components/NotesPage/modal/AddNewNoteCard.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { ColorPaletteIcon } from '../../../assets/svg/allsvg.jsx';
+import { MyEditor } from '../MyEditor.jsx';
+import '../NotesPage.css';
+function AddNewNoteCard({
+	formData,
+	onEditorStateChange,
+	editorState,
+	submitNote,
+	titleChangeHandler,
+	tagChangeHandler,
+	priorityChangeHandler,
+	colorChangeHandler,
+}) {
+	const { title, body, tag, priority, cardColor, createdAt } = formData;
+	const isBodyEmpty = editorState.getCurrentContent().getPlainText().length === 0;
+	const canSubmit = !isBodyEmpty && title.length > 0;
```

Comment:
I am still able to submit the note even though the button is disabled. 
![image](https://user-images.githubusercontent.com/46674359/161894802-4abdf27f-95c1-4019-8e80-646d66a07418.png)
Suggestion:
Add pointer events: none to prevent this.

Reply:
Done fixed